src/ansys/geometry/core/tools/problem_areas.py

```diff
@@ -619,3 +625,45 @@ def fix(self) -> RepairToolMessage:
         message = RepairToolMessage(response.result.success, [], [])
 
         return message
+
+
+class LogoProblemArea(ProblemArea):
+    """Represents a logo problem area defined by a list of faces.
+
+    Parameters
+    ----------
+    id : str
+        Server-defined ID for the problem area.
+    grpc_client : GrpcClient
+        Active supporting geometry service instance for design modeling.
+    faces : list[str]
+        List of faces defining the logo problem area.
+    """
+
+    def __init__(self, id: str, grpc_client: GrpcClient, face_ids: list[str]):
+        """Initialize a new instance of the logo problem area class."""
+        super().__init__(id, grpc_client)
+
+        self._face_ids = face_ids
+
+    @property
+    def face_ids(self) -> list["str"]:
```

Comment:
We might not need the quotes around str as it is a built-in type.
```suggestion
    def face_ids(self) -> list[str]:
```

Reply:
Fixed. Thx :-)